doc/index.md

```diff
@@ -1,100 +1,178 @@
 # Introduction
 
-lua-http is an HTTP library for Lua, it supports: both client and server operations, both HTTP 1 and HTTP 2.
+Lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. Lua-http can be run in either a Server (listen) or Client (connect) mode and includes the following features: 
+- HTTP 1 and HTTP 2 as specified by RFCs [1945](https://tools.ietf.org/html/rfc1945) and [7540 ](https://tools.ietf.org/html/rfc7540)
+- WebSockets as specified by RFCs [6455](https://tools.ietf.org/html/rfc6455) and [7936](https://tools.ietf.org/html/rfc7936) including ping/pong, binary data transfer and TLS encyrption
```

Comment:
heh, I didn't even know about RFC 7936.... looks like I got it correct by accident ;)